### PR TITLE
Rails 6.1 support

### PR DESCRIPTION
**Pull request by jamesbrooks**

### CODE REVIEW

**Comment by kvokka** [comment]
This branch does **not** test against rails 6.1 in CI, see #1013

File: lib/acts-as-taggable-on.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 require 'active_record'
 require 'active_record/version'
 require 'active_support/core_ext/module'
+require 'active_support/core_ext/string/starts_ends_with'
```
Comment (silva96) [inline lib/acts-as-taggable-on.rb:4]: why is this needed?

Comment (jamesbrooks) [reply lib/acts-as-taggable-on.rb:4]: Replacing `ends_with?` with `end_with?` would had also been an alternative, that require was to bring in that alias.

**Comment by jamesbrooks** [comment]
@kvokka thanks, addressed in 786bd25

**Comment by jamesbrooks** [comment]
Closing in favor of #1013 which resolves additional issues
